cpp/daal/include/services/internal/error_handling_w_a.h

```diff
@@ -0,0 +1,39 @@
+/* file: error_handling_w_a.h */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef __SERVICES_INTERNAL_ERROR_HANDLING_WORKAROUND_H__
+#define __SERVICES_INTERNAL_ERROR_HANDLING_WORKAROUND_H__
+
+namespace daal
+{
+namespace services
+{
+
+namespace interface1
+{
+    class Status;
+} // namespace interface1
+
+namespace internal
+{
+    ErrorID get_error_id(interface1::Status &s);
+} // namespace internal
```

Comment:
Probably it is better to move function declaration and entire `internal` namespace inside of `namespace interface1`? In that case, you don't have to specify namespaces for function parameter in this file and in error_handling.h